.github/workflows/userbenchmark-regression-detector.yml

```diff
@@ -24,33 +24,12 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: benchmark
-      - name: Create conda environment
-        run: |
-          . "${SETUP_SCRIPT}"
-          python benchmark/utils/python_utils.py --create-conda-env "${CONDA_ENV_NAME}"
-          sudo python benchmark/utils/cuda_utils.py --setup-cuda-softlink
       - name: Tune Nvidia GPU
         run: |
           . "${SETUP_SCRIPT}"
           sudo nvidia-smi -pm 1
           sudo nvidia-smi -ac 1215,1410
           nvidia-smi
-      - name: Install PyTorch nightly
```

Comment:
This means you are never checking the PyTorch nightly version in the docker even if it is staled.
Say if pytorch nightly fails to build for 10 days, you will keep running this workflow for 10 days on the same pytorch version. Just want to confirm is this desired behavior?

Reply:
Yes. I'd still want signal since it is relatively early for the optim benchmarks so if regressions happen it would be good to note regardless of the pytorch nightly.